core/modules/parser/parseTreeUtil.cc

```diff
@@ -45,12 +45,12 @@ class DigraphVisitor {
     typedef std::map<AnAst, std::string> AstMap;
 
     DigraphVisitor() : i(0) {}
-    void operator()(AnAst a, C& p) {
+    void operator()(AnAst const &a, C const & p) {
```

Comment:
Somewhat inconsistent whitespace use in reference declarations. I looked quickly at qserv code, most frequently references are declared as `Class& ref` or `Class const& ref`, very rarely as `Class &ref`. `Class const & ref` hass one additional space. I think it's better to stick to most common use.